Add ability to change datepicker icon (#6494)
laravel-filament/filament: packages/forms/docs/03-fields.md

use Filament\Forms\Components\DateTimePicker;

DateTimePicker::make('date')

>     ->icon('heroicon-o-calendar')

Reviewer: This snippet should remain unchanged?

    ->disabledDates(['2022-10-02', '2022-10-05', '2022-10-15'])
```

> The calendar icon can be changed, or disabled by passing `icon(false)`

Reviewer: Maybe add a separate example below for disabling.
Also add `:` to line before example.

    ->disabledDates(['2022-10-02', '2022-10-05', '2022-10-15'])
```

> The calendar icon can be changed:

Reviewer: ```suggestion
You may change the calendar icon using the `icon()` method:
```

```php
use Filament\Forms\Components\DateTimePicker;

DateTimePicker::make('date')
    ->icon('heroicon-o-calendar')
```

> Or you may pass `false` to hide the icon:

Reviewer: ```suggestion
Alternatively, you may remove the icon altogether by passing `false`:
```
